diff --git a/arch/arm64/boot/dts/freescale/s32v234.dtsi b/arch/arm64/boot/dts/freescale/s32v234.dtsi
index b0b0747..dd238e3 100644
--- a/arch/arm64/boot/dts/freescale/s32v234.dtsi
+++ b/arch/arm64/boot/dts/freescale/s32v234.dtsi
@@ -111,6 +111,18 @@
 			reg = <0x0 0xFD000000 0x0 0x400000>;    /* 4 MB */
 			no-map;
 		};
+
+		resmem: rmem@C4F00000 {
+			reg = <0 0xC4F00000 0 0x100000>;
+			no-map;
+		};
+	};
+
+	nxpresmem: themem@C4F00000 {
+		status = "okay";
+		compatible = "nxp,resmem";
+		memory-region = <&resmem>;
+		interrupts = <0 0 4>;
 	};
 
 	timer {
